service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -278,4 +295,42 @@ void getDatasetPreviewTable() {
         datasetPreviewTable.getColumns().get(0),
         is(new bio.terra.catalog.model.ColumnModel().name("column a").arrayOf(false)));
   }
+
+  @Test
+  void testExportDatarepoDataset() {
+    reset(datasetDao);
```

Comment:
I don't think this is necessary, as I'd expect `@Mock` objects to be re-initialized on every test run. Does it work without `reset()`? Looking at the code, this is also done in `testGetMetadataUsingTdrPermission()` but I think that use is also unnecessary and should've been removed when `mockDataset()` was added.

Reply:
It does work without reset, I've removed those.